dashboard/app/views/lti/v1/integration.html.haml

```diff
@@ -0,0 +1,39 @@
+#signup
```

Comment:
TODO: currently I'm borrowing some css classes from the signup page, which isn't ideal.